opm/input/eclipse/Schedule/RSTConfig.hpp

```diff
@@ -204,10 +204,12 @@ class RSTConfig
     RSTConfig() = default;
     RSTConfig(const SOLUTIONSection& solution_section,
               const ParseContext& parseContext,
+              const bool compositional,
```

Comment:
It is possible that we can make `bool compositional` a member of `RSTConfig` to save the interface changes.